charts/karpenter-crd/values.yaml

```diff
@@ -1,7 +0,0 @@
-webhook:
```

Comment:
We can probably delete this `values.yaml`

Reply:
Yep, I deleted it.